src/nvme/fabrics.c

```
@@ -282,7 +282,7 @@ static int inet6_pton(const char *src, uint16_t port,
 
 	char  *tmp = strdup(src);
 	if (!tmp)
-		nvme_msg(LOG_ERR, "cannot copy: %s\n", src);
+		return -ENOMEM;		// Just logging here looks like a bug
```

Yes, this should just return the error code.